src/Cron/DayOfWeekField.php

```diff
@@ -36,7 +36,10 @@ public function isSatisfiedBy(\DateTime $date, $value)
             $tdate = clone $date;
             $tdate->setDate($currentYear, $currentMonth, $lastDayOfMonth);
             while ($tdate->format('w') != $weekday) {
```

Comment:
Since `$tdate` isn't updated correctly and keeps the same value, this while loop never ends.